src/Uno.UI/UI/Xaml/FontFamily.cs

```diff
@@ -42,7 +45,7 @@ public override bool Equals(object obj)
 				return true;
 			}
 
-			return !ReferenceEquals(a, null) && a.Equals(b);
+			return a is not null && a.Equals(b);
```

Comment:
I think that this equivalent to `a.Equals(null)` which is not the same as previous (as we have override the `Equals`). Regarding the current implementation the result will be the same (unless `Equals` is overridden in a sub-class), but even if less cute, I would keep `ReferenceEquals` as it's more performant ... unless that behavior change is expected of course!

(BTW personally when I override `Equals` implement equality operators, I prefer to create a static method `Equals(MyType left, MyType right)` which use only `Equals` and/or `ReferenceEquals`, and redirect all other to this one. This avoid unexpected recursive calls.)

Reply:
`c is null` is equvalient to `object.ReferenceEquals(c, null)`